lib/portage/_emirrordist/MirrorDistTask.py

```diff
@@ -161,7 +154,7 @@ def _scheduled_deletion_log(self):
             date = timestamp + deletion_delay
             if date < start_time:
                 date = start_time
-            date = time.strftime("%Y-%m-%d", time.gmtime(date))
+            date = time.strftime("%s-%s-%s", time.gmtime(date))
```

Comment:
Ah, I think I know what happened there. I was doing codebase-wide sed regex substitution for `%\l` patterns. While I was traversing the modifications, I saw that line and thought I needed to revert them back to `%s` from instinct because it was a commonly used Old-Style format token.

This change needs to be reverted.